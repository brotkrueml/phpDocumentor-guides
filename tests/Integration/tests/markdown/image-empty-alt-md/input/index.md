# Images with and without alt text

![with alt Text](https://example.org/example.png)

![](https://example.org/example2.png)
