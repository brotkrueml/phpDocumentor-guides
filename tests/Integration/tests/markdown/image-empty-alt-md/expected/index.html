<!DOCTYPE html>
<html lang="en">
<head>
    <title>Images with and without alt text - Project Title</title>

</head>
<body>
    <h1>Images with and without alt text</h1>

<p><img src="https://example.org/example.png" alt="with alt Text"/></p>
<p><img src="https://example.org/example2.png" alt=""/></p>
</body>
</html>
